src/main/java/com/commercetools/sync/inventories/README.md

```diff
@@ -1,287 +1,122 @@
-entry to keep it up to date. To receive such
-[UpdateActions](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-sdk-base/src/main/java/io/sphere/sdk/commands/UpdateAction.java)
-you can use following methods:
-
-#### `buildChangeQuantityAction`
-Used for comparision of `quantityOnStock` values from provided entries. `Optional` that may contain updating
-[ChangeQuantity](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-models/src/main/java/io/sphere/sdk/inventory/commands/updateactions/ChangeQuantity.java)
-action is returned.
+**Preconditions:** The sync expects a list of `InventoryEntryDraft` objects that have their `sku` fields set.
+Also every `InventoryEntryDraft`, that belongs to a `Channel`, in your input list must either:
+- Has the [Reference](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-sdk-base/src/main/java/io/sphere/sdk/models/Reference.java)
+to the `supplyChannel` expanded. This means that calling `getObj()` on the reference would not
+return `null`, but returns the [Channel](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-models/src/main/java/io/sphere/sdk/channels/Channel.java)
+object, from which the sync can access its `key`. Example of sync performed that way can be found [here](https://github.com/commercetools/commercetools-sync-java/blob/master/src/integration-test/java/com/commercetools/sync/inventories/InventorySyncTest.java#L128).
+- If the reference is not expanded, then it is very important to provide the channel `key` in the `id` field of the
+reference. This means calling `getId()` on the channel reference would return it's `key`. Example of sync performed that
+way can be found [here](https://github.com/commercetools/commercetools-sync-java/blob/master/src/integration-test/java/com/commercetools/sync/inventories/InventorySyncTest.java#L181).
 
-````java
-//Having an old InventoryEntry attempts to build "change quantity" update action
-final Optional<UpdateAction<InventoryEntry>> updateAction =
-    InventoryUpdateActionUtils.buildChangeQuantityAction(inventoryEntry, inventoryEntrydraft);
-
-// If update action is present, the old entry can be updated by the SphereClient instance.
-if (updateAction.isPresent()) {
-    sphereClient.execute(InventoryEntryUpdateCommand.of(inventoryEntry, updateAction.get()));
-}
-````
+<!-- TODO: Ensure above paths to examples in a dedicated branch -->
 
-#### `buildSetRestockableInDaysAction`
-Used for comparision of `restockableInDays` values from provided entries. `Optional` that may contain updating
-[SetRestockableInDays](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-models/src/main/java/io/sphere/sdk/inventory/commands/updateactions/SetRestockableInDays.java)
-action is returned.
+The sync results in a `ComplectionStage` that contains `InventorySyncStatistics` object. From a received instance
```

Comment:
`This object contains all the stats of the sync process: a report message, the total number of updated, created, failed, processed categories and the processing time of the sync in different time units and in a human readable format.`